docs/docs/getting-started.md

```diff
@@ -41,9 +41,63 @@ $ npm install --save react react-dom babel-preset-react
 $ webpack
 ```
 
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+Ensure you have a `.babelrc` including the react preset.
+
+A good starting webpack config for production mode would be:
+```javascript
+module.exports = {
+  entry: [
+    './client/index.jsx',
+  ],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+  },
+  resolve: {
+    modulesDirectories: ['node_modules', 'client'],
```

Comment:
I know I'm late to this, but `['node_modules']` is the default anyway, so that whole `resolve: {}` block could be removed (presuming `.jsx` is dropped in favour for `.js`), making the config less overwhelming for a 'getting started, and the webpack docs covers this config and many others in depth.